tensorflow/tensorflow.bzl

```diff
@@ -111,6 +111,7 @@ def tf_copts():
                 "/DPLATFORM_WINDOWS",
                 "/DEIGEN_HAS_C99_MATH",
                 "/DTENSORFLOW_USE_EIGEN_THREADPOOL",
+		"/DEIGEN_VECTORIZE_SSE3",
```

Comment:
Add `// To flush denormals without __SSE3__ set`.  Terrifyingly obscure otherwise.